mono/mini/debugger-agent.c

```diff
@@ -1189,6 +1218,35 @@ socket_transport_accept (int socket_fd)
 	
 	return conn_fd;
 }
+static void 
+create_file_to_communicate_port (int sfd, MonoError* error)
+{
+	struct sockaddr_in sin;
+	socklen_t len = sizeof(sin);
+	if (getsockname (sfd, (struct sockaddr *)&sin, &len) != -1) {
+		char* file_name = g_strdup_printf ("debugger_attach.%d", getpid());
+		char* full_path = g_build_filename (g_get_tmp_dir (), file_name, (const char*)NULL);
```

Comment:
Nit: `char *full_path`

Reply:
g_get_tmp_dir () might return a value sdb/ide might not know about i.e. if somebody sets the TEMP env var.